pkg/handlers/v1/eni_test.go

```diff
@@ -24,7 +24,7 @@ func TestTransformENI(t *testing.T) {
 			InputFile: "eni.create.json",
 			ExpectedOutput: Output{
 				AccountID:    "123456789",
-				ChangeTime:   "2020-08-21T12:00:00.000Z",
+				ChangeTime:   "2020-08-14T21:00:00.000Z",
```

Comment:
question: why did we need to change the time here? did the our changes here effect how this test would behave?

Reply:
The timestamp was changed to be the ENI's attachTime, instead of the ConfigurationItemCaptureTime, as part of the shift away from using the capture time.